pkg/commands/commands.go

```diff
@@ -68,6 +68,8 @@ func GetCommand(cmd instructions.Command, buildcontext string) (DockerCommand, e
 		return &ArgCommand{cmd: c}, nil
 	case *instructions.ShellCommand:
 		return &ShellCommand{cmd: c}, nil
+	case *instructions.HealthCheckCommand:
+		return &HealthCheckCommand{cmd: c}, nil
```

Comment:
From the build error, it looks like you'll have to translate from the dockerfile HealthCheck type to the go-containerregistry one.

Reply:
exactly. Just looking into how to do this the most efficiently. 

Just creating a v1.HealthConfig and passing all values would work, but I wouldn't really regard this as future proof or clean tbh. Any alternatives you could think of?